style.scss

```diff
@@ -1596,7 +1596,7 @@ body >footer {
           width: 45%;
           position: relative;
           box-shadow: 0 1px 3px rgba(0,0,0,0.2);
-
+          
```

Comment:
Not a showstopper, but: you have added unnecessary trailing whitespace; see:

- http://programmers.stackexchange.com/questions/121555/why-is-trailing-whitespace-a-big-deal

- https://www.gnu.org/software/emacs/manual/html_node/emacs/Useless-Whitespace.html

For your editor, you can often remove this kind of "diff noise". See:

- http://stackoverflow.com/questions/30403150/atom-text-editor-remove-trailing-whitespace-on-save

- "Whitespace Normalizer" as discussed at https://johnpapa.net/my-recommended-brackets-extensions/